diff --git a/django_extensions/management/commands/runserver_plus.py b/django_extensions/management/commands/runserver_plus.py
index f380515..16d8ca6 100644
--- a/django_extensions/management/commands/runserver_plus.py
+++ b/django_extensions/management/commands/runserver_plus.py
@@ -181,6 +181,16 @@ class Command(BaseCommand):
             if open_browser:
                 import webbrowser
                 webbrowser.open(bind_url)
+
+            import errno, socket
+            from werkzeug.serving import BaseWSGIServer
+            def handle_error(self, *args, **kwargs):
+                exc_type, exc_value = sys.exc_info()[:2]
+                if exc_type is socket.error and exc_value.errno == errno.EPIPE:
+                    return
+                super(BaseWSGIServer, self).handle_error(*args, **kwargs)
+            BaseWSGIServer.handle_error = handle_error
+
             if cert_path:
                 """
                 OpenSSL is needed for SSL support.
